photon-server/src/main/java/org/photonvision/raspi/PicamJNI.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2020 Photon Vision.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.photonvision.raspi;
+
+import java.nio.file.Path;
+import org.photonvision.common.hardware.Platform;
+import org.photonvision.common.logging.LogGroup;
+import org.photonvision.common.logging.Logger;
+
+public class PicamJNI {
+
+    private static boolean isSupported = false;
+    private static Logger logger = new Logger(PicamJNI.class, LogGroup.Camera);
+
+    static {
+        try {
+            System.load(
+                    Path.of("/opt/vc/src/hello_pi/photon-picam-driver/libpicam.so")
```

Comment:
Does the `.so` have to be here? Can it instead go in the `/opt/photonvision` directory?

Reply:
We now extract the shared object from the jar to `./lib/` when we're on the Pi. This will extract to `/opt/photonvision` in most cases.